src/test/java/com/thirdcc/webapp/security/jwt/RefreshTokenProviderTest.java

```diff
@@ -0,0 +1,109 @@
+package com.thirdcc.webapp.security.jwt;
+
+import com.thirdcc.webapp.config.ApplicationProperties;
+import com.thirdcc.webapp.security.AuthoritiesConstants;
+import io.jsonwebtoken.Jwts;
+import io.jsonwebtoken.SignatureAlgorithm;
+import io.jsonwebtoken.io.Decoders;
+import io.jsonwebtoken.security.Keys;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.springframework.security.authentication.UsernamePasswordAuthenticationToken;
+import org.springframework.security.core.Authentication;
+import org.springframework.security.core.GrantedAuthority;
+import org.springframework.security.core.authority.SimpleGrantedAuthority;
+import org.springframework.test.util.ReflectionTestUtils;
+
+import java.security.Key;
+import java.util.ArrayList;
+import java.util.Collection;
+import java.util.Date;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+public class RefreshTokenProviderTest {
```

Comment:
I saw all is negative test
no need positive test?

Reply:
since you requested. Added in 601389fb0b0ae00d290752ab1470394e258a5862